src/reactviews/pages/ConnectionDialog/azureBrowsePage.tsx

```diff
@@ -1,9 +1,334 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the MIT License. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the MIT License. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { useContext, useEffect, useState } from "react";
+import { ConnectionDialogContext } from "./connectionDialogStateProvider";
+// import { FormField } from "../../common/forms/form.component";
+import { ConnectButton } from "./components/connectButton.component";
+import { Dropdown, Field, Option, Button } from "@fluentui/react-components";
+import { FormField, useFormStyles } from "../../common/forms/form.component";
+import { FormItemSpec } from "../../common/forms/form";
+import { IConnectionDialogProfile } from "../../../sharedInterfaces/connectionDialog";
+import { AdvancedOptionsDrawer } from "./components/advancedOptionsDrawer.component";
+import { locConstants } from "../../common/locConstants";
+import { TestConnectionButton } from "./components/testConnectionButton.component";
+
+function removeDuplicates<T>(array: T[]): T[] {
+    return Array.from(new Set(array));
+}
+
+export const AzureBrowsePage = () => {
+    const context = useContext(ConnectionDialogContext);
+    const formStyles = useFormStyles();
+    const [isAdvancedDrawerOpen, setIsAdvancedDrawerOpen] = useState(false);
+
+    if (context === undefined) {
+        return undefined;
+    }
+
+    const [subscriptions, setSubscriptions] = useState<string[]>([]);

```

Comment:
You can probably combine these states into a single object or you can move these into the dropdown component. That will make the code simpler. 

So Azure dropdown will have 2 states variable  `values` and `value`.